### PR TITLE
US104330: Move some behavior imports around.

### PR DESCRIPTION
**Pull request by njostonehouse**

### CODE REVIEW

File: src/d2l-my-courses-content/d2l-my-courses-content-behavior.js [hunk 1]
```diff
@@ -725,10 +725,9 @@ D2L.MyCourses.MyCoursesContentBehaviorImpl = {
 * @polymerBehavior D2L.MyCourses.MyCoursesContentBehavior
 */
 D2L.MyCourses.MyCoursesContentBehavior = [
-	D2L.MyCourses.MyCoursesBehavior,
-	D2L.MyCourses.CourseTileResponsiveGridBehavior,
```
Comment (njostonehouse) [inline src/d2l-my-courses-content/d2l-my-courses-content-behavior.js]: This is only used in segregated view.

File: src/d2l-my-courses-content/d2l-my-courses-content-behavior.js [hunk 2]
```diff
@@ -725,10 +725,9 @@ D2L.MyCourses.MyCoursesContentBehaviorImpl = {
 * @polymerBehavior D2L.MyCourses.MyCoursesContentBehavior
 */
 D2L.MyCourses.MyCoursesContentBehavior = [
-	D2L.MyCourses.MyCoursesBehavior,
```
Comment (njostonehouse) [inline src/d2l-my-courses-content/d2l-my-courses-content-behavior.js]: This was only included for localization.

File: src/d2l-my-courses-content/d2l-my-courses-content-animated.js [hunk 3]
```diff
@@ -132,7 +132,8 @@ Polymer({
 		}
 	},
 	behaviors: [
-		D2L.MyCourses.MyCoursesContentBehavior
+		D2L.MyCourses.MyCoursesContentBehavior,
+		D2L.MyCourses.CourseTileResponsiveGridBehavior
```
Comment (edwardg8) [inline src/d2l-my-courses-content/d2l-my-courses-content-animated.js]: Is this new added behavior used anywhere?

Comment (edwardg8) [reply src/d2l-my-courses-content/d2l-my-courses-content-animated.js]: Oh, i see, never mind.

Comment (njostonehouse) [reply src/d2l-my-courses-content/d2l-my-courses-content-animated.js]: The `CourseTileResponsiveGridBehavior` used to be added to the `MyCoursesContentBehavior`, but it's only needed here.

**Comment by njostonehouse** [comment]
Test failures galore. Forgot to check tests, in favour of running the demo >_< sorry.

**Pull request closed without merging**